src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -42,6 +42,9 @@ public class DatabasesLandlord : IDisposable
         private readonly SemaphoreSlim _databaseSemaphore;
         private readonly ServerStore _serverStore;
 
+        // used in ServerWideBackupStress
+        internal static bool SkipShouldContinueDisposeCheck = false;
```

Comment:
I would prefer this to not be 'static'. You can get instance of RavenServer in tests and access DatabasesLandlord there.

Reply:
done